### PR TITLE
TA-1 Added the Definition of  Git Strategy Workflow 

### PR DESCRIPTION
**Pull request by ecaterinamihaluta**
Added gitworkflow package and Gitworklowclass

### CODE REVIEW

**Review by AndreiSecu — APPROVED** [review]

**Review by AndreiSecu — CHANGES_REQUESTED** [review]
There are missing some steps we discussed last time, please add them and then perform git ammend

File: src/main/java/org/ecaterina/gitworkflow/GitWorkflow.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+     System.out.println(step4);
+     System.out.println(step5);
+     System.out.println(step6);
+     System.out.println(step7);
+     System.out.println(step8);
+     System.out.println(step9);
+     System.out.println(step10);
+     System.out.println(step11);
+     System.out.println(step12);
+
```
Comment (AndreiSecu) [inline src/main/java/org/ecaterina/gitworkflow/GitWorkflow.java]: you have to print step13

**Review by AnastasiaNeghina — APPROVED** [review]

File: src/main/java/org/ecaterina/gitworkflow/GitWorkflow.java (hunk 1 above)
Comment (GalburG) [reply src/main/java/org/ecaterina/gitworkflow/GitWorkflow.java]: step13 could be also in english ;) 
and actually you need to do commit amend first and then force push , pls add this 
+ after step3 you need to add git pull also , or just add it in step3

**Review by GalburG — CHANGES_REQUESTED** [review]

**Comment by GalburG** [comment]
@ecaterinamihaluta  still need to add 'çommit amend' copy the last step pls

**Review by AndreiSecu — APPROVED** [review]

**Review by AndreiSecuQA — APPROVED** [review]

**Comment by GalburG** [comment]
Ecaterina, step 19 pls remove it , useless now there

**Pull request merged**